Content.Server/GameTicking/Rules/Configurations/NinjaRuleConfiguration.cs

```diff
@@ -0,0 +1,66 @@
+using Content.Server.Objectives;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.List;
+
+namespace Content.Server.GameTicking.Rules.Configurations;
+
+/// <summary>
+/// Configuration for the Space Ninja antag.
+/// </summary>
+public sealed class NinjaRuleConfiguration : StationEventRuleConfiguration
+{
+    /// <summary>
+    /// List of objective prototype ids to add
+    /// </summary>
+    [DataField("objectives", required: true, customTypeSerializer: typeof(PrototypeIdListSerializer<ObjectivePrototype>))]
+    public readonly List<string> Objectives = new();
+
+    // TODO: move to job and use job???
```

Comment:
like you have here, you can just do this on the job. pretty sure you already did and just commented it out, fwiw.